v3/v3_service_test.go

```diff
@@ -5809,3 +5809,67 @@ func TestOperations_ListCluster(t *testing.T) {
 	assert.Equal(t, "cluster", *prismElements[0].Metadata.Kind)
 	assert.Equal(t, "ganon", prismElements[0].Status.Name)
 }
+
+func TestOperations_CreateIdempotenceIdentifers(t *testing.T) {
+	creds := testhelpers.CredentialsFromEnvironment(t)
+	interceptor := khttpclient.NewInterceptor(http.DefaultTransport)
+	v3Client, err := NewV3Client(creds, WithRoundTripper(interceptor))
+	require.NoError(t, err)
+
+	kctx := mock.NewContext(mock.Config{
+		Mode: keploy.MODE_RECORD,
```

Comment:
Once, you're done recording, you have to switch this back to test mode..

Reply:
Thanks, I'll fix that. I did it once, but then had to re-record, and forgot the second time :laughing:

Reply:
Done in https://github.com/nutanix-cloud-native/prism-go-client/pull/204/commits/4e2cb80834d25165c784bad8a01242892fc1f06f